content/SCALE/SCALEUIReference/SystemSettings/Services/FTPServiceScreenSCALE.md

```diff
@@ -31,80 +31,79 @@ To configure FTP, go to **System Settings > Services** and find **FTP**, then cl
+| **Timeout** | Enter the maximum client idle time in seconds before disconnecting. The default value is **600** seconds. |
 
 ## FTP Advanced Settings
-**Advanced Settings** include the **General Options** on the **Basic Settings** configuration screen, and allow you to specify access permissions, TLS settings, bandwidth and other setting to further customize FTP access.
+**Advanced Settings** include the **General Options** on the **Basic Settings** configuration screen and allow you to specify access permissions, TLS settings, bandwidth, and other settings to customize FTP access.
 
 ### Access and TLS Setting Options
 
 ![FTPAdvancedSettingsAccess](/images/SCALE/22.12/FTPAdvancedSettingsAccess.png "Services FTP Advanced Settings Access Options")
 
 #### Access Settings 
-**Access** settings specify user login, file and directory access permissions.
+**Access** settings specify user login, file, and directory access permissions.
 
 | Settings | Description |
 |----------|-------------|
-| **Always Chroot** | Select to only allow users access their home directory if they are in the **wheel** group. This option increases security risk. To confine FTP sessions to a home directory of a local user, enable **chroot** and select **Allow Local User Login**. |
-| **Allow Root Login** | Select to allow root logins. This option increases security risk so enabling this is discouraged. Do *not* allow anonymous or root access unless it is necessary. 
-For better security, enable TLS when possible (especially when exposing FTP to a WAN). TLS effectively makes this [FTPS](https://tools.ietf.org/html/rfc4217). |
-| **Allow Anonymous Login** | Select to allow anonymous FTP logins with access to the directory specified in **Path**. Selecting this displays the **Path** field. Enter or browse to the loction to populate the field. |
-| **Allow Local User Login** | Select to allow any local user to log in. By default, only members of the **ftp** group are allowed to log in. |
+| **Always Chroot** | Only allows users to access their home directory if they are in the **wheel** group. This option increases security risk. To confine FTP sessions to a local user home directory, enable **chroot** and select **Allow Local User Login**. |
+| **Allow Root Login** | Select to allow root logins. This option increases security risk, so enabling this is discouraged. Do *not* allow anonymous or root access unless it is necessary. 
+Enable TLS when possible (especially when exposing FTP to a WAN). TLS effectively makes this [FTPS](https://tools.ietf.org/html/rfc4217) for better security. |
+| **Allow Anonymous Login** | Select to allow anonymous FTP logins with access to the directory specified in **Path**. Selecting this displays the **Path** field. Enter or browse the location to populate the field. |
+| **Allow Local User Login** | Select to allow any local user to log in. Only members of the **ftp** group may log in by default. |
```

Comment:
not sure we should use may because we are not granting permission to do this, rather saying they can do this by default.
```suggestion
| **Allow Local User Login** | Select to allow any local user to log in. Only members of the **ftp** group can log in by default. |
```